device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClientDiagnostic.java

```diff
@@ -0,0 +1,85 @@
+package com.microsoft.azure.sdk.iot.device;
+
+import java.text.DecimalFormat;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * Handling device client diagnostic information.
+ */
+public class DeviceClientDiagnostic {
+    private int diagSamplingPercentage;
+    private int currentMessageNumber;
+
+    private static final String DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
```

Comment:
seems no use

Reply:
forget to delete this since dateformat is changed, fixed